lib/helpers.js

```diff
@@ -72,3 +72,17 @@ exports.generateNonce = async ({entryStyle = 'human'} = {}) => {
   throw new Error(`Invalid "entryStyle" "${entryStyle}"; "entryStyle" must ` +
     `be "human" or "machine".`);
 };
+
+exports.combinedHash = ({clientId, hash} = {}) => {
+  if(clientId) {
+    return exports.prefixedHash(`${hash}:${clientId}`);
+  } else {
+    return exports.prefixedHash(hash);
+  }
+};
+
+exports.verifyHash = ({clientId, hash, sha256} = {}) => {
+  return crypto.timingSafeEqual(
+    new Buffer(sha256, 'base64'),
+    new Buffer(exports.combinedHash({clientId, hash}), 'base64'));
```

Comment:
I see this was copy/paste from existing code, however, I believe this syntax generates a deprecation warning like: 
```
(node:32) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
```
I think you'll want to use `Buffer.from`.